rdr_service/api/participant_summary_api.py

```diff
@@ -3,20 +3,29 @@
 
 from rdr_service.api.base_api import BaseApi, make_sync_results_for_request
 from rdr_service.api_util import AWARDEE, DEV_MAIL, RDR_AND_PTC, PTC_HEALTHPRO_AWARDEE_CURATION
-from rdr_service.app_util import auth_required, get_validated_user_info
+from rdr_service.app_util import auth_required, get_validated_user_info, restrict_to_gae_project
 from rdr_service.dao.base_dao import _MIN_ID, _MAX_ID
+from rdr_service.dao.participant_dao import ParticipantDao
 from rdr_service.dao.participant_summary_dao import ParticipantSummaryDao
 from rdr_service.model.hpo import HPO
 from rdr_service.model.participant_summary import ParticipantSummary
 from rdr_service.config import getSettingList, HPO_LITE_AWARDEE
 from rdr_service.code_constants import UNSET
 from rdr_service.participant_enums import ParticipantSummaryRecord
 
+PTC_ALLOWED_ENVIRONMENTS = [
+    'all-of-us-rdr-sandbox',
+    'all-of-us-rdr-stable',
+    'all-of-us-rdr-ptsc-1-test',
+    'localhost'
```

Comment:
Do we want to add  `all-of-us-rdr-ptsc-2-test` and `all-of-us-rdr-ptsc-3-test` ?

Reply:
Good question. Just going to keep the same pattern/list as what is being used in the OutReach API POST.
https://github.com/all-of-us/raw-data-repository/blob/devel/rdr_service/api/genomic_api.py#L14